openarc/openarc.c

```diff
@@ -3257,7 +3257,7 @@ mlfi_eom(SMFICTX *ctx)
 				       afc->mctx_jobid, AR_HEADER_NAME);
```

Comment:
Please change the logged message to end with "; continuing" or "; ignoring" or something like that.